### PR TITLE
Hungarian translation

### PR DESCRIPTION
**Pull request by gerazo**

### CODE REVIEW

File: po/hu.po [hunk 1]
```diff
@@ -1619,24 +1830,30 @@ msgid ""
...
+"Részfeladatok:\n"
+"%s"
 
 #: GTG/plugins/send_email/sendEmail.py:61
 #, python-format
 msgid ""
 "\n"
 "Task content:\n"
-"%s"
+""
```
Comment (Neui) [inline po/hu.po:1842]: Changed source string which you shouldn't do.
https://github.com/getting-things-gnome/gtg/blob/27b04d3c86a7217ccae712ee0b7fa9891f12858c/GTG/plugins/send_email/sendEmail.py#L61
Note that the translated version also lacks the "%s" part.

**Review by Neui — COMMENTED** [review]
Found a small mistake, easy to fix. 
I don't speak Hungarian, so I can't check the translation itself.

File: po/hu.po (hunk 1 above)
Comment (gerazo) [reply po/hu.po:1842]: Huhh. Thanks. Nice catch.

Comment (gerazo) [reply po/hu.po]: Fixed. (Not other similar was found.)

**Comment by diegogangl** [comment]
@gerazo How do you want to move forward with this? Do you have someone who knows Hungarian that you could invite to review? Or should I go ahead and merge?

Files look good

**Comment by kristofking007** [comment]
I checked the translation, it looks fine to me. ( Hungarian is my first language. )

**Comment by YuKaiTe** [comment]
I just checked too, translation looks good. (Also native Hungarian speaker)

**Comment by diegogangl** [comment]
Merged, thanks guys!

**Comment by gerazo** [comment]
Thanks a lot for everyone.

**Pull request merged**